src/tasks/AppleAppBuilder/AppleAppBuilder.cs

```diff
@@ -167,10 +166,76 @@ public string TargetOS
     /// </summary>
     public bool StripSymbolTable { get; set; }
 
+    /// <summary>
+    /// Bundles the application for NativeAOT runtime. Default runtime is Mono.
+    /// </summary>
+    public bool UseNativeAOTRuntime { get; set; }
+
+    public void ValidateRuntimeSelection()
+    {
+        if (UseNativeAOTRuntime)
+        {
+            if (!string.IsNullOrEmpty(MonoRuntimeHeaders))
+            {
+                Log.LogMessage(MessageImportance.High, $"Property \"{nameof(MonoRuntimeHeaders)}\" is not supported with NativeAOT runtime and will be ignored.");
```

Comment:
I think we can make all of these errors instead of ignoring.

Reply:
Unsupported properties converted to exceptions.